src/reducers/copy.js

```diff
@@ -39,50 +39,81 @@ export default (pattern) => (base, source) => {
+    if (!compareTagsOnly) {
+      debug(`testing`);
+        const filterMissing = function(sourceField) {
+          // Test 02: Identical control fields are not copied
+          if ('value' in sourceField) {
+            return baseFields.some(isIdenticalControlField) === false;
+          }
+          // Test 04: Identical data fields in base and source, not copied
+          // Test 05: Different data fields are copied from source to base
+          if ('subfields' in sourceField) {
+            return baseFields.some(isIdenticalDataField) === false;
+          }
 
-      function isIdenticalControlField(baseField) {
-        const normalizedBaseField = normalizeControlField(baseField);
-        const normalizedSourceField = normalizeControlField(sourceField);
-        return normalizedSourceField === normalizedBaseField;
-      }
-      function isIdenticalDataField(baseField) {
-        if (sourceField.tag === baseField.tag &&
-          sourceField.ind1 === baseField.ind1 &&
-          sourceField.ind2 === baseField.ind2 &&
-          sourceField.subfields.length === baseField.subfields.length) {
-          return baseField.subfields.every(isIdenticalSubfield);
-        }
-        function normalizeSubfield(subfield) {
-          return subfield.value.toLowerCase().replace(/\s+/u, '');
-        }
-        function isIdenticalSubfield(baseSub) {
-          const normBaseSub = normalizeSubfield(baseSub);
-          return sourceField.subfields.some(sourceSub => {
-            const normSourceSub = normalizeSubfield(sourceSub);
-            return normSourceSub === normBaseSub;
-          });
+          function normalizeControlField(field) {
+            return field.value.toLowerCase().replace(/\s+/u, '');
+          }
+
+          function isIdenticalControlField(baseField) {
+            const normalizedBaseField = normalizeControlField(baseField);
+            const normalizedSourceField = normalizeControlField(sourceField);
+            return normalizedSourceField === normalizedBaseField;
+          }
+          function isIdenticalDataField(baseField) {
+            if (sourceField.tag === baseField.tag &&
+              sourceField.ind1 === baseField.ind1 &&
+              sourceField.ind2 === baseField.ind2 &&
+              sourceField.subfields.length === baseField.subfields.length) {
+              return baseField.subfields.every(isIdenticalSubfield);
+            }
+            function normalizeSubfield(subfield) {
+              return subfield.value.toLowerCase().replace(/\s+/u, '');
+            }
+
+            function isIdenticalSubfield(baseSub) {
+              const normBaseSub = normalizeSubfield(baseSub);
+                // If excluded subfields have been defined for this field
+                if(excludeSubfields) {
+                  debug(`in excludeSubfields block`);
+                  const excludedSubfields = [...excludeSubfields]; // make array from string
```

Comment:
Should not be a string. Should be an array of strings which is passed to reducer creation function.